badges/badgesIcon.css

```diff
@@ -0,0 +1,52 @@
+:root{
+    --badgeIconGreyCol:grey;
+    --badgeBackgoundColor:rgb(218, 207, 207);
+}
+*{
+    padding:0;
+    margin:0;
+    box-sizing: border-box;
+    font-family: sans-serif;
+    color:black;
+
+}
+.badgeIcons{
+    width:30%;
+    display:flex;
+    flex-direction: row;
+    gap:2rem;
+    padding:1rem;
+}
+.badgeIcon{
+    font-family: sans-serif;
+    color:black;
+    position: relative;
+    
+}
+.badge-icon1{
+    font-size: 3rem;
+    color:var(--badgeIconGreyCol);
+}
```

Comment:
Here class names are similar. May get confusing later. Also use similar naming style: either use camelCase or kebab-case. Not both.